mypy/typeanal.py

```diff
@@ -944,9 +944,9 @@ def analyze_callable_type(self, t: UnboundType) -> Type:
                 )
                 if maybe_ret is None:
                     # Callable[?, RET] (where ? is something invalid)
-                    # TODO(PEP612): change error to mention paramspec, once we actually have some
```

Comment:
Or Concatenate?

Reply:
Though that's getting a bit much for an error message.

Reply:
I like the new language, may a note linking to some docs would be useful in addition however.

Reply:
Yeah, Concatenate is why I went with the more generic "parameter specification" rather than "ParamSpec". This error is probably disproportionately encountered by new users, so don't want it to be too scary.

I could add a note for https://mypy.readthedocs.io/en/stable/kinds_of_types.html#callable-types-and-lambdas